### DISCUSSION TITLE
[Idle notifier] Add churning and milking cows to notify when done

### DISCUSSION THREAD

**Discussion by Hecu93** [runelite/runelite#17310] [Ideas]
I noticed the plugin doesn't seem to recognize these two activities and it would make it a bit more enjoyable to not be relying on your hearing only when these two activities are done as they take pretty long to finish in the first place.

**Comment by Felanbird** [comment]
@Hecu93 Just so you're aware these were added in 4cf320d50f77480e2ee991bded5e7b51888a31ac & 259d92c63242d63477a05a95498735e7c4d50bed and is fixed in the RuneLite version that came out a few hours ago
